prowler/providers/aws/aws_provider.py

```diff
@@ -64,27 +64,50 @@ class AwsProvider(Provider):
     # TODO: this is not optional, enforce for all providers
     audit_metadata: Audit_Metadata
 
-    def __init__(self, arguments: Namespace):
+    def __init__(
```

Comment:
I think it is a good time to rename the arguments to something better like:
- `input_role` -> `role_arn`
- `input_external_id` -> `external_id`

Reply:
Done!